### ISSUE TITLE
Some bug with 0 universe

### ISSUE THREAD

**Issue by timurash** [Xaver2k/ArduinoENC_Artnet-WS2811#3]
I did everything as you explained, but i have some bug that i cannot solve. Second universe going really good, but first universe not showing up correctly. I tried to change pins, number of universe, but nothing helps. To more precise: somehow, on first universe first pixel always on with green color, and other pixels react on changes, but not correctly. 

Do you have any suggestions that may help me solve this problem?

**Comment by Xaver2k** [comment]
Hi, 
did you try switching pins from universe 1 and 2 with each other ?
Do you use the supplied libraries? The normal Adafruit_NeoPixel library will not work correctly

**Comment by timurash** [comment]
Yes, i tried switching pins from universe 0 and 1 with each other. The problem is still remains on universe 0.

Yes, i'm using modified NeoPixel library

**Comment by Xaver2k** [comment]
Did you switch led strip to be sure the LEDs are working properly?

**Comment by timurash** [comment]
Yes, i switched led strip. Nothing helps

**Comment by Xaver2k** [comment]
Do you have a direct ethernet connection from your pc to the arduino or do you use a switch or router in-between?
If so please try a direct connection or switch ethernet cables.
Programmatically there is not really a difference from first and second universe. You could try using universe 3 and 4 instead of 0 and 1

**Comment by timurash** [comment]
First I tried to connect arduino direct into my pc, then i tried make connection through router. Still no result. Second universe doing great, but first universe continues messing around

I also tried change my config to 3 and 4 universes, but I got the same result

**Comment by Xaver2k** [comment]
hmm kind of weird, do you use the same UIPEthernet library, or tested different Arduino and ENC Ethernet module?

**Comment by Brezzel** [comment]
I was having exactly the same problem universe 1 flawless but 0 was going nuts. I have found a fix though.

 Change from this 
#​define​ ​DATAPIN1​ ​2​  ​//​ 
 ​#​define​ ​DATAPIN2​ ​4​  ​//


To this
#​define​ ​DATAPIN1​ ​4  ​//​ 
 ​#​define​ ​DATAPIN2​ ​8  ​//​


Still don't have a clue what pin's 5/7 do or how to connect them but I have 2 universes working without them. If anyone knows let me know.
